src/inthandler.S

```diff
@@ -190,10 +197,10 @@ interrupt:
 	and t1, t0
 	sw t1, STACK_SR(sp)
 
-	# Reload cause register (might be reused by C code) and test for other interrupts
+	# Reload cause register (might be reused by C code (TODO: what?)) and test for other interrupts
```

Comment:
oh I didn't understand it that way at all
would "clobbered" instead of "reused" be fine?
```suggestion
	# Reload cause register (might be clobbered by C code) and test for other interrupts
```
or even
```suggestion
	# Reload cause register (might be clobbered by external handlers) and test for other interrupts
```